docs/operators/jdbcPollInputOperator.md

```diff
@@ -0,0 +1,174 @@
+
+#### Dynamic Partitioning
+Not supported.
+
+## Operator Information
+1. Operator location: ***malhar-library***
+2. Available since: ***3.5.0***
+3. Operator state: ***Evolving***
+4. Java Packages:
+    * Operator: ***[com.datatorrent.lib.db.jdbc.AbstractJdbcPollInputOperator](https://www.datatorrent.com/docs/apidocs/com/datatorrent/lib/db/jdbc/AbstractJdbcPollInputOperator.html)***
+
+JDBC Poller is **idempotent**, **fault-tolerant** and **statically partitionable**.
+
+## AbstractJdbcPollInputOperator
+This is the abstract implementation that serves as base class for polling messages from jdbc store. It can be extended to modify functionality or add new capabilities. This class doesn’t have any ports, so concrete subclasses will need to provide them if necessary.
+
+![AbstractJdbcPollInputOperator.png](images/jdbcinput/operatorsClassDiagram.png)
+
+###<a name="AbstractJdbcPollInputOperatorProps"></a>Properties of AbstractJdbcPollInputOperator
+Several properties are available to configure the behavior of this operator and they are summarized in the table below.
+
+| **Property** | **Description** | **Type** | **Mandatory** | **Default Value** |
+| -------- | ----------- | ---- | ------------------ | ------------- |
+| *store* | JDBC Store for connection | [JDBCStore](#JDBCStore) | Yes | N/A |
+| *tableName* | table name to be scanned | String | Yes | N/A |
+| *columnsExpression* | Comma separated list of columns to select from the given table. | String | Yes | N/A |
+| *key* | Primary key column name | String | Yes | N/A |
+| *partitionCount* | Static partitions count | int | No | 1 |
+| *whereCondition* | Where condition for jdbc query | String | No | N/A |
+| *fetchSize* | Hint limiting the number of rows to fetch in a single call | int | No | 20000 |
+| *pollInterval* | Interval in milliseconds to poll the database table | int | No | 10000 |
+| *queueCapacity* | Capacity of queue which holds DB data before emiting | int | No | 4096 |
+| *batchSize* | Maximum number of tuples to emit in a single call to the emitTuples() callback (see explanation above). | int | No | 2000 |
+
+#### <a name="JDBCStore"></a>Properties of JDBC Store (BackendStore)
+| **Property** | **Description** | **Type** | **Mandatory** | **Default Value** |
+| -------- | ----------- | ---- | ------------------ | ------------- |
+| *databaseDriver* |JDBC Driver class for connection to JDBC Store. This driver should be there in class path | String | Yes | N/A |
+| *databaseUrl* | Database url of the form jdbc:subprotocol:subname | String | Yes | N/A |
+| *connectionProps* | Comma separated connection properties e.g. user:xyz,password:ijk | String | Yes | N/A |
+
+Of these only `store`, `tableName`, `columnsExpression` and `key` are mandatory. Those properties can be set like this:
```

Comment:
Actually store object is set in java code. Should I add that comment at the end of properties section?